students/kmsnyde/lesson2/series.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Apr  3 15:31:18 2018
+
+@author: Karl M. Snyder
+"""
+
+def fibonacci(n):
+    """Creates a fibonacci sequence with the number of elements determined
+    by the "n" parameter"""
+    
+    list = []
+    for i in range(n):
+        if (i == 0) or (i == 1):
+            list.append(i)
+        else:
+            list.append(list[-1] + list[-2])
+    print(*list, '\n')
+    return list
+fibonacci(20)
+
+def lucas(n):
+    """The Lucas Numbers are a related series of integers that start with
+    the values 2 and 1.  Returns the nth value in the lucas numbers series."""
+    
+    list = [2, 1]
+    for i in range(n-2):
+        list.append(list[-1] + list[-2])
+    print(*list, '\n')
+    return list
+lucas(20)
+
+def sum_series(count, start1=0, start2=1):
+    """Creates a sequence of length "count", in which the first 2 elements
+    default to 0 and 1, or which may be changed to any 2 numbers when calling
+    the function using start1/start2 argument positions."""
+    
+    if start1 == 0 and start2 == 1:
+        list = []
```

Comment:
avoid using built in names because it will overwrite the built-in. notice github has a special syntax highlighting for `list`